### ISSUE TITLE
On Combobox Narrator is not reading as required even if aria-required="true" is set ,as aria-required attribute goes to parent hierarchy instead of <input> control 

### ISSUE THREAD

**Issue by v-ajrai** [microsoft/fluentui#15890]
On Combobox Narrator is not reading as required even if aria-required="true" is set ,as aria-required attribute goes to parent hierarchy instead of <input> control .
### Environment Information

OS: Win10
Browser: Edge chromium Version 87.0.634.0 (Official build) dev (64-bit),

### Describe the issue:

On Combobox Narrator is not reading as required even if aria-required="true" is set ,as aria-required attribute goes to parent hierarchy instead of <input> control .

### Please provide a reproduction of the issue in a codepen:

https://codepen.io/v-ajrai/pen/dyXgxKN?editable=true%3Dhttps%3A%2F%2Fdeveloper.microsoft.com%2F
![combo_box_external](https://user-images.githubusercontent.com/62233982/98645894-9cd17900-2358-11eb-8c76-3ff862cfb412.PNG)

#### Actual behavior:
On Combobox Narrator is not reading as required even if aria-required="true" is set ,as aria-required attribute goes to parent hierarchy instead of <input> control .

#### Expected behavior:

On Combobox Narrator should read as required when aria-required="true" is set and aria required attribute should be able set on <input>

**Comment by paulgildea** [comment]
@v-ajrai Thanks for filing this issue. We'll take a look.

**Comment by v-ajrai** [comment]
@paulgildea  Could you please let us know when we could get the fix for it.

**Comment by v-ajrai** [comment]
any update on this ...?

**Comment by v-ajrai** [comment]
@paulgildea Could you please let us know when we could get the fix for it.

**Comment by annaowens** [comment]
@v-ajrai I ran into a similar issue and was able to get around this by setting `required={true}` on the Combo Box element. The `required` tag automatically added `aria-required` to the element itself and was read by the screen reader.

(My team was using a custom element that inherited from Combo Box, so I also had to add `required={this.props.required}` to the custom element so that the props would work)
